cmd/fluentd/docker/conf/loki.conf

```diff
@@ -0,0 +1,9 @@
+<match loki.**>
+  @type loki
+  url "#{ENV['LOKI_URL']}"
+  extra_labels {"env":"dev"}
```

Comment:
maybe job="fluentd" is a better default ?

Reply:
Done in 244f196